src/mmw/apps/user/models.py

```diff
@@ -32,3 +34,41 @@ class ItsiUser(models.Model):
 
     def __unicode__(self):
         return unicode(self.user.username)
+
+
+class UserProfile(models.Model):
+    UNSPECIFIED = 'Unspecified'
+    UNI_FACULTY = 'University Faculty'
+    UNI_PROFESSIONAL = 'University Professional or Research Staff'
+    POST_DOC = 'Post-Doctoral Fellow'
+    UNI_GRAD_STU = 'University Graduate Student'
+    UNI_UGRAD_STU = 'University Undergraduate Student'
+    COMMERCIAL = 'Commercial/Professional'
+    GOVERNMENT = 'Government Official'
+    K_12_STU = 'School Student Kindergarten to 12th Grade'
+    K_12_TEACHER = 'School Teacher Kindergarten to 12th Grade'
+    OTHER = 'Other'
+
+    USER_TYPE_CHOICES = (
+        (UNSPECIFIED, UNSPECIFIED),
+        (UNI_FACULTY, UNI_FACULTY),
+        (UNI_PROFESSIONAL, UNI_PROFESSIONAL),
+        (POST_DOC, POST_DOC),
+        (UNI_GRAD_STU, UNI_GRAD_STU),
+        (UNI_UGRAD_STU, UNI_UGRAD_STU),
+        (COMMERCIAL, COMMERCIAL),
+        (GOVERNMENT, GOVERNMENT),
+        (K_12_STU, K_12_STU),
+        (K_12_TEACHER, K_12_TEACHER),
+        (OTHER, OTHER),
+    )
+
+    user = models.OneToOneField(User, primary_key=True)
+    was_skipped = models.BooleanField(default=False)
```

Comment:
Will this always be the negation of `is_complete`?

Reply:
For now, yes, but here is the workflow where they will not be:

- Log in for the first time
- Click "...later...". `was_skipped` is set to true.
- Using the feature in #2256 fill out the form. `is_complete` is set to true.